spring-boot/proxy/src/main/java/eu/coatrack/proxy/metrics/MetricsCounterService.java

```diff
@@ -88,7 +88,7 @@ private String createCounterIdFromMetric(TemporaryMetricsAggregation tma) {
                 .add(tma.getApiKeyValue())
                 .add(tma.getMetricType().toString())
                 .add(String.valueOf(tma.getHttpResponseCode()))
-                .add(LocalDate.now().toString())
+                .add(java.sql.Date.valueOf(LocalDate.now()).toString())
```

Comment:
This change does not seem to have any effect on the String that is eventually created, or does it? If yes, please explain.

Reply:
You are right. Therefore, I reverted that change.